x/gov/README.md

```diff
@@ -320,7 +349,7 @@ which is modifiable by governance. This means that proposals are accepted if:
 
 #### No inheritance
 
-If a delegator does not vote, it won't inherit its validator vote.
+If a delegator does not vote, his validators will not inherit his vote.
```

Comment:
```suggestion
If a delegator does not vote, the vote of the delegated validator - if applicable - will not be inherited.
```

Reply:
there's a mistake here, the delegator inherits the validator vote not the other way around